module/VuFind/src/VuFind/RecordDriver/SolrMarcRemote.php

```diff
@@ -32,7 +32,7 @@
 
 namespace VuFind\RecordDriver;
 
-use Laminas\Log\LoggerAwareInterface as LoggerAwareInterface;
+use Psr\Log\LoggerAwareInterface as LoggerAwareInterface;
```

Comment:
This can still be simplified:

```suggestion
use Psr\Log\LoggerAwareInterface;
```